data/actions/scripts/other/potions.lua

```diff
@@ -143,11 +145,12 @@ function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 		end
 		
 		player:addCondition(exhaust)
-		target:say("Aaaah...", TALKTYPE_MONSTER_SAY)
-		
-		if potions.removepots then
+		if potions.sayWords and potions.words then
+			target:say("Aaaah...", TALKTYPE_MONSTER_SAY)
```

Comment:
I think you meant `target:say(potions.words, TALKTYPE_MONSTER_SAY)`, but the condition could be simplified to `if potions.words then` and you remove `sayWords` entirely :grinning: